src/transaction/log_recovery.c

```diff
@@ -5013,6 +5028,65 @@ log_recovery_undo (THREAD_ENTRY * thread_p)
 	}
     }
 
+  /* Find log_Gl m_active_start position located in the last GROUP COMPLETE record before m_min_active_lsa (min pos */
+  /* of last active transaction at the end of analysis) - we go backwards until we find a GC record */
+  if (!LSA_ISNULL (&log_Gl.m_min_active_lsa))
+    {
+      assert (log_Gl.m_active_start_position == 0);
+      bool found = false;
+      LSA_COPY (&log_lsa, &log_Gl.m_min_active_lsa);
+      while (!found)
+	{
+	  if (logpb_fetch_page (thread_p, &log_lsa, LOG_CS_FORCE_USE, log_pgptr) != NO_ERROR)
```

Comment:
What if the GROUP_COMPLETE is in the log archive and the archive was removed?

Reply:
added todo & opened an issue